src/Microsoft.VisualStudio.Services.Agent/Constants.cs

```diff
@@ -228,5 +229,17 @@ public static class Task
                 public static readonly string DisplayName = "task.displayname";
             }
         }
+
+        public class Release
+        {
+            public class Path
+            {
+                public static readonly string WorkingDirectory = "r";
```

Comment:
Another option would be to prefix your folders with "r" instead of creating an "r" subdirectory. E.g. `r1`, `r2`.

This would eliminate introducing confusion over which directory is the work directory. "_work" *is the work directory* and existing variables and configuration say the same thing.